bottles/frontend/ui/details-preferences.ui

```diff
@@ -509,7 +509,7 @@
                         <property name="subtitle" translatable="yes">Use a restricted/managed environment for this bottle.</property>
                         <child>
                             <object class="GtkButton" id="btn_manage_sandbox">
-                                <property name="tooltip-text" translatable="yes">Manage Sandbox permissions</property>
+                                <property name="tooltip-text" translatable="yes">Manage the sandbox permissions</property>
```

Comment:
I think the original `Manage sandbox permissions`  was already correct

Reply:
it was, but I think it would be best to have it as `Manage the sandbox permissions`. I can revert the change, if you'd like.

Reply:
I think `Manage the sandbox permissions` isn't grammatically correct, I don't think that there should be a `the` in front of the `sandbox`

Reply:
GNOME Circle asked to Capitalize All Titles

Reply:
Got some input from an external person, I'll capitalize it to be in accordance with the HIG and for proper grammar.

Reply:
Except some words like "a", "the", etc.